examples/ccd-js-gen/client-tokenMetadata.ts

```diff
@@ -0,0 +1,74 @@
+import { credentials } from '@grpc/grpc-js';
+import * as SDK from '@concordium/node-sdk';
+import meow from 'meow';
+import { parseEndpoint } from '../shared/util';
+
+//import * as wCCDModule from './lib/wCCD';
```

Comment:
What is this?

Reply:
I have added a comment explaining why I left this here